lib/watch.js

```diff
@@ -15,9 +15,7 @@ module.exports = function (gulp, options) {
   var opts = merge({}, defaults, options);
   var watchers = opts.watchers;
 
-  gulp.task('watch', function () {
-    watchers.forEach(function (item) {
-      gulp.watch(item.match, item.tasks);
-    });
+  gulp.task('watch', () => {
+    watchers.forEach(item => gulp.watch(item.match, item.tasks));
```

Comment:
Curious about linting/style patterns when it comes to single-line anonymous callbacks. I find this line a tad hard to read but it may be my naivete WRT ES2015.